vanilla-framework/vanilla-framework
# Update docs/tooltips page content

## Done
- Updated copy ✏️ 
- Updated demo example 💯 
- Check content matches copy doc [here](https://docs.google.com/document/d/1n_FNUUnBt3Mnka_c6w3lp2ov84IpC8p0Na6t7THP16s/edit#heading=h.o3hx1ll2z8d7)

## QA
- Pull code
- Run /docs > ./run
  - Check http://0.0.0.0:8104/en/patterns/tooltips
- Pull code
- Run ./run
- Review example
  - Check http://0.0.0.0:8101/vanilla-framework/examples/patterns/tooltips/

## Screenshot
![Screenshot 2019-04-04 at 16 36 10](https://user-images.githubusercontent.com/17748020/55568560-d27f9c00-56f7-11e9-8b69-1acda1f545c7.png)
